tools/checked-c-convert/ProgramInfo.h

```diff
@@ -96,6 +99,13 @@ class ConstraintVariable {
   }
 
   virtual ~ConstraintVariable() {};
+
```

Comment:
Could you add comments that describe what these functions do?

Reply:
Yes.